### PR TITLE
feat: add risk properties support to csv-to-oscal-cd task

### LINKED ISSUE #1544: Add RISK support in to component definition spread sheet (csv) support
## Issue description / feature objectives

Add the following optional columns as expected but optional to the csv-to-oscal-cd task:

- $Original_Risk_Rating
    - Provide the Original Risk Rating from the scanner
- $Adjusted_Risk_Rating
    - Provide the Adjusted Risk Rating as approved by the CIO
- $Risk_Adjustment
    - Whether there was a Risk Adjustment

These values, if specified, are to be added to the resulting OSCAL component definition as Properties.

## Caveats / Assumptions

Provide sample/test/tutorial csv's with these additional fields/descriptions.

## Completion Criteria

Test cases for code coverage added.

### PR DESCRIPTION
**Pull request by Ma1h01**
## Types of changes

- [ ] Hot fix (emergency fix and release)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Documentation (change which affects the documentation site)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Release (`develop` -> `main`)

## Quality assurance (all should be covered).

- [x] My code follows the code style of this project.
- [x] Documentation for my change is up to date?
- [x] My PR meets testing requirements.
- [x] All new and existing tests passed.
- [x] All commits are signed-off.

## Key links:

Resolved #1544
My approach to this issue is as follows:

In `csv-to-oscal-cd.py`:

1. Add three constants to represent the three risk headings.
2. Add the constants to the `_columns_optional` and `_rule_property_column_names` lists because they are optional and not required when component type is validation.
3. Modify the `print_info()` to correctly show the three newly added optional headings.

Then, make a new csv file with all three risk headings added for testing.

In `csv-to-oscal-cd-test.py`:
1. Write one test case to verify the presence of the three risk props in the resulting `component-definition.json`.
2. Write one test case to verify the absence of the three risk props in the resulting `component-definition.json`, when the component type is validation. 

# Before you merge

- Ensure it is a 'squash commit' if not a release.
- Ensure CI is currently passing
- Check sonar. If you are working for a fork a maintainer will reach out, if required.

### CODE REVIEW

File: tests/data/csv/bp.sample.v5.csv
Comment (degenaro) [inline tests/data/csv/bp.sample.v5.csv:1]: Why is this new file needed? Can the 3 new column headings and associated data be mocked easily instead? Or, can you add the 3 column headings to an existing file, where all existing tests pass? And mock just the data for these added optional columns in the relevant new test case(s)?

Comment (Ma1h01) [reply tests/data/csv/bp.sample.v5.csv:1]: Great point! I removed the new csv file and mocked the optional columns in the tests instead.

**Review by degenaro — APPROVED** [review]
LGTM

**Review by AleJo2995 — APPROVED** [review]
LGTM

**Comment by AleJo2995** [comment]
@Ma1h01 you can go ahead and squash and merge this one

**Pull request merged**